import_map.json

```diff
@@ -2,7 +2,7 @@
   "imports": {
     "deno-slack-sdk/": "https://deno.land/x/deno_slack_sdk@2.3.0/",
     "deno-slack-api/": "https://deno.land/x/deno_slack_api@2.1.2/",
-    "std/": "https://deno.land/std@0.202.0/",
+    "std/": "https://deno.land/std@0.183.0/",
```

Comment:
Was this intentional, going back to a previous version?

Reply:
🤔 I didn't make the PR but the github action is using my credentials to it shows I'm the author, maybe @hello-ashleyintech can answer the question

Reply:
👋 I noticed this in a few other samples too! This seems like [a difficult one to keep up with](https://github.com/denoland/deno_std/releases), but we might want to bump this to the latest (`0.203.0`) for [all samples using it](https://github.com/slackapi/deno-sample-dependencies/blob/main/unique_dependencies_2/import_map.json) for now?